ZAPDTR/ZAPD/Main.cpp

```diff
@@ -434,6 +477,8 @@ int main(int argc, char* argv[])
 		exporterSet->endProgramFunc();
 
 end:
+	delete exporterSet;
+	//Globals::Instance->GetExporterSet() = nullptr; TODO NULL this out
```

Comment:
is this something that needs to happen? if not
```suggestion
```

Reply:
At some point it should IMO because there are checks to see if its null.

Reply:
why isn't it working now? why does it need to be commented out?